### PR TITLE
Rename message length headers and set payload max to 255

### PR DESCRIPTION
**Pull request by jp-bennett**
As per Semtech's documentation, the payload max length is actually 255 bytes. https://www.mouser.com/datasheet/2/761/sx1276-1278113.pdf page 65

### CODE REVIEW

**Comment by fifieldt** [comment]
Guess what? With this, my SX1276 Tbeam can now send messages to the LR1110-based T-1000E.

**Review by fifieldt — APPROVED** [review]
This makes sense to me, and based on a small amount of testing appears to improve things for my devices.

However, will leave merge to people who are more versed in radio things ;)

**Comment by jp-bennett** [comment]
> Guess what? With this, my SX1276 Tbeam can now send messages to the LR1110-based T-1000E.

Wait what? ... what?

**Comment by fifieldt** [comment]
Also, T114 doesn't crash now.

**Review by thebentern — APPROVED** [review]

File: src/mesh/RadioInterface.h [hunk 1]
```diff
@@ -9,8 +9,9 @@
 
 #define MAX_TX_QUEUE 16 // max number of packets which can be waiting for transmission
 
-#define MAX_RHPACKETLEN 256
-#define LORA_HEADER_LENGTH 16
+#define MAX_LORA_PAYLOAD_LEN 255 // max length of 255 per Semtech's datasheets on SX12xx
+#define MESHTASTIC_HEADER_LENGTH 16
```
Comment (GUVWAF) [inline src/mesh/RadioInterface.h:13]: Shall we redefine this to `sizeof(PacketHeader)` or at least change the assert in `RadioInterface::RadioInterface()` to `MESHTASTIC_HEADER_LENGTH`?

**Review by GUVWAF — COMMENTED** [review]
Good find!

**Pull request merged**